minecraft_servers/voidroleplay/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "server_name": "voidroleplay",
+  "nice_name": "VoidRoleplay",
+  "direct_ip": "voidroleplay.de",
+  "supported_languages": [
+     "de"
+  ],
+  "social": {
+    "web": "https://voidroleplay.de",

```

Comment:
Website in maintenance mode;
If it still takes some time, remove

Reply:
Alright. Removed!